worker/uniter/remotestate/watcher.go

```diff
@@ -146,14 +148,16 @@ func NewWatcher(config WatcherConfig) (*RemoteStateWatcher, error) {
 		// so that we coalesce events while the observer is busy.
 		out: make(chan struct{}, 1),
 		current: Snapshot{
-			Relations:      make(map[int]RelationSnapshot),
-			Storage:        make(map[names.StorageTag]StorageSnapshot),
-			ActionsBlocked: config.ContainerRunningStatusChannel != nil,
-			ActionChanged:  make(map[string]int),
+			Relations:           make(map[int]RelationSnapshot),
+			Storage:             make(map[names.StorageTag]StorageSnapshot),
+			ActionsBlocked:      config.ContainerRunningStatusChannel != nil,
+			ActionChanged:       make(map[string]int),
+			UpgradeSeriesStatus: model.UpgradeSeriesNotStarted,
```

Comment:
what's the upgrade series change for?

Reply:
Because UpgradeSeriesStatus normally defaults to UpgradeSeriesNotStarted in IAAS because of initial events on remote state, but because CAAS doesn't use this, the reboot resolver was checking for it but UpgradeSeriesStatus was empty string.